framework/LoopMeUnitedSDK/Core/Internal/Utility/LoopMeIdentityProvider.m

```diff
@@ -104,4 +104,14 @@ + (NSNumber *)customAuthorizationStatus {
     }
 }
 
++ (CGFloat)calculatePxratioOfScreen {
+    CGFloat screenWidth = [[UIScreen mainScreen] bounds].size.width;
```

Comment:
I think this calculation is just w/h. This looks like it calculates Pixels per square inch and divides by 1000.